dev/com.ibm.ws.jdbc_fat_db2/publish/servers/com.ibm.ws.jdbc.fat.db2/server.xml

```diff
@@ -52,8 +57,17 @@
       commandTimeout="2m30s" connectionTimeout="90s" memberConnectTimeout="100000ms"/>
   </dataSource>
   
+    <dataSource jndiName="jdbc/db2jar">
```

Comment:
Not quite sure what the purpose of this extra datasource is, `jdbc/db2` already uses `properties.db2.jcc` and thus does not need to infer which Datasource class to use.